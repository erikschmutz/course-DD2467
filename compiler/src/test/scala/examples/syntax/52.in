let i = 1..0 +.. 2..0