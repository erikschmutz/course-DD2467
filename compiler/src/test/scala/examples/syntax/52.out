Program(
    List(
        Assignment(
            Identifier(i),
            OperatorExpr(
                DoubleLit(1.0),
                DoubleLit(2.0),
                DoublePlus()
            )
        )
    )
)